concepts/05 UI Components/DateRangeBox/00 Getting Started with DateRangeBox/10 Set the Accepted Date Range.md

```diff
@@ -0,0 +1,106 @@
+You can define an available range from which end-users can select dates. For this purpose, specify the [min](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#max) properties:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#dateRangeBox").dxDateRangeBox({
+            min: new Date(1920, 0, 1),
```

Comment:
may be we shiuld set more strict man/max? With values like this user won't see how disabling dates less than min/bigger than max working. How about +- two weeks from current date?

Reply:
Ok, I think it is good idea too